src/Microsoft.AspNetCore.Dispatcher/Resources.resx

```diff
@@ -164,7 +164,7 @@
     <value>The route parameter name '{0}' is invalid. Route parameter names must be non-empty and cannot contain these characters: '{{', '}}', '/'. The '?' character marks a parameter as optional, and can occur only at the end of the parameter. The '*' character marks a parameter as catch-all, and can occur only at the start of the parameter.</value>
   </data>
   <data name="TemplateRoute_InvalidRouteTemplate" xml:space="preserve">
-    <value>The route template cannot start with a '/' or '~' character.</value>
+    <value>The route template cannot start with a '~' character.</value>
```

Comment:
This is a bit misleading, the route template can start with `~` if its followed by a `/`

Reply:
`The route template cannot start with a '~' character unless followed by a "/".`

Is this any better?